src/raphael_backend_flask/templates/multimodal_analysis.html

```diff
@@ -0,0 +1,107 @@
+{% extends 'base.html' %}
```

Comment:
I suspect I’d be tempted to refactor this so that there’s a new analysis.html, that the two analysis templates extend from (because the first bit and last bit of both templates is the same).

But that may very well be a bit silly, and could be more confusing than is really necessary. Maybe it’s fine to have some duplication here.

Reply:
Actually, I think this is fine.